### PR TITLE
add first etag  and treesize propagation tests and steps

### PR DESCRIPTION
**Pull request by micbar**
# Description

## Add

- [x] Etag Propagation Tests
- [x] TreeSize Propagation tests

### CODE REVIEW

File: features/etag-propagation.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: Etag propagation
+  As as a oCIS user
+  I want to be edit files in my directory tree
```
Comment (butonic) [inline features/etag-propagation.feature]: > I want to be edit?

**Review by butonic — APPROVED** [review]

File: featurecontext/featurecontext.go [hunk 2]
```diff
@@ -14,6 +14,11 @@ type User struct {
 	User      *userv1beta1.User
 }
 
+type ResourceAlias struct {
+	Ref *providerv1beta1.Reference
+	Info *providerv1beta1.ResourceInfo
```
Comment (wkloucek) [inline featurecontext/featurecontext.go:19]: I would prefer to not store information that can change rather quickly (eg. mtime). Doing a stat when actually inspecting the resource shouldn't be that bad?

Comment (micbar) [reply featurecontext/featurecontext.go:19]: I want to do a stat and compare checksums and etags with the previos values.

File: features/etag-propagation.feature (hunk 1 above)
Comment (micbar) [reply features/etag-propagation.feature]: fixed

**Pull request merged**